server/db/init.js

```diff
@@ -1,32 +1,30 @@
 import knexModule from 'knex';
 import bookshelfModule from 'bookshelf';
 
-const knex = knexModule({
-  client: 'pg',
-  connection: process.env.PG_CONNECTION_STRING,
-  debug: true,
-});
+const initializeDB = function(config, module) {
+  module(config);
+}
 
-const bookshelf = bookshelfModule(knex);
+const bookshelf = function(bookshelfObj) {
+  bookshelfObj.knex.schema.hasTable('users').then((exist) => {
+    if (!exist) {
+      bookshelfObj.knex.schema.createTableIfNotExists('users', (table) => {
+        table.increments();
+        table.string('email').notNullable().unique().index();
+        table.string('first_name').notNullable();
+        table.string('last_name').notNullable();
+        table.string('phone_number').notNullable();
+        table.string('password_hash').notNullable();
+        table.boolean('is_admin').defaultTo(false);
+        table.boolean('is_banned').defaultTo(false);
+        table.boolean('is_active').defaultTo(true);
+        table.timestamp('date_created').defaultTo(knex.fn.now());
+        table.timestamp('date_updated').defaultTo(knex.fn.now());
+      }).then(() => {
+        console.log(('Created users table'));
+      });
+    }
+  });
+}
 
-bookshelf.knex.schema.hasTable('users').then((exist) => {
-  if (!exist) {
-    bookshelf.knex.schema.createTableIfNotExists('users', (table) => {
-      table.increments();
-      table.string('email').notNullable().unique().index();
-      table.string('first_name').notNullable();
-      table.string('last_name').notNullable();
-      table.string('phone_number').notNullable();
-      table.string('password_hash').notNullable();
-      table.boolean('is_admin').defaultTo(false);
-      table.boolean('is_banned').defaultTo(false);
-      table.boolean('is_active').defaultTo(true);
-      table.timestamp('date_created').defaultTo(knex.fn.now());
-      table.timestamp('date_updated').defaultTo(knex.fn.now());
-    }).then(() => {
-      console.log(('Created users table'));
-    });
-  }
-});
-
-export default bookshelf;
+export default { bookshelf, initializeDB };
```

Comment:
@nguyenaiden says we're going to remove initializeDB from this export.

make a decision about whether or not to export the renamed 'bookshelf' function as default or in its own named export.